cypress/integration/dashboard/events.js

```diff
@@ -161,6 +161,38 @@ describe('events dashboard', () => {
     cy.mhDeleteAll();
   }
 
+  it('invitation email should include calendar file', () => {
+    cy.visit('/dashboard/events/1');
+
+    cy.findByRole('button', { name: 'Email Attendees' }).click();
+
+    // try to make sure there will be recipient
+    cy.findByLabelText('Waitlist').click({ force: true });
+    cy.findByLabelText('Cancelled').click({ force: true });
```

Comment:
Do we need to force, here? It's a bit of an indication that something's not quite right.

Reply:
This is the matter in few other tests in `events.js` as well, for some reason cypress detects those elements as hidden by another element, and without forcing, it doesn't work. I didn't dig too deep to find what's up with that.

Reply:
Understood. I'll add that to my backlog, then.